loopd/config.go

```diff
@@ -299,6 +306,16 @@ func Validate(cfg *Config) error {
 		return fmt.Errorf("must specify --lnd.macaroonpath")
 	}
 
+	// Allow at most 1 hour timeout.
```

Comment:
nit: comment + error out of date now that we've bumped the default, I'd just go with "at most double our sane default" + fill the value in the error